app/jobs/create_organization_report_job.rb

```diff
@@ -0,0 +1,33 @@
+class CreateOrganizationReportJob < ApplicationJob
+  queue_as :default
+
+  def perform(id:)
+    organization = Organization.find(id)
+    return unless organization
```

Comment:
`find` will raise an error if we don't find the record. You may want to change to `find_by id: id` so this job doesn't fail and reenque.